src/app/account/login/login.component.ts

```diff
@@ -58,6 +65,31 @@ export class LoginComponent implements OnInit, OnDestroy {
             this.errorMessage = LOGIN_ERROR_MESSAGES[error] || LOGIN_ERROR_MESSAGES.internalError;
         }
     }
+    onSubmit() {
+        const LoginData = {
```

Comment:
We should name constants with lower camel case.
```suggestion
        const loginData = {
```

Reply:
done.